gnmi_server/client_subscribe.go

```diff
@@ -119,6 +121,8 @@ func (c *Client) Run(stream gnmipb.GNMI_SubscribeServer) (err error) {
 	var dc sdc.Client
 	if target == "OTHERS" {
 		dc, err = sdc.NewNonDbClient(paths, prefix)
+	} else if target == "SONiC_DB" {
```

Comment:
could we change this into "SONIC_DB". It will be error-prone using the logo string with "i" as the DB name.

Reply:
Renamed.